meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```
     return (
       <div className="App">
       {
         <div>
```
What is this for? Do you need this curly brace? I think that's only for javascript inside of your jsx element